### PR TITLE
feature/COR-1380-missing-sanity-keys

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard26**
## Summary

* updated 'Positive tests' pages on NL and VR to receive GGD graph titles from Sanity rather than hardcoded values;
* added Sanity keys for NL and VR;
* updated graphs to use new toggle logic introduced as part of COR-1275 for consistency purposes;
* cleaned up because of consistency changes;

### Sanity
Sanity lokalize keys can be found in the following path inside Sanity sudio: `Dashboard Pagina's > Positieve testen > nl > ggd.linechart_totaltests_toggle_label OR ggd.linechart_percentage_toggle_label`.

### Screenshots
Screenshots cover only the NL scope, yet these changes have also been applied for VR. GM does not contain these toggle-able graphs. Screenshots show the Sanity key paths.

#### Before
image

#### After
image

### CODE REVIEW

File: packages/app/src/pages/landelijk/positief-geteste-mensen.tsx [hunk 1]
```diff
@@ -1,27 +1,24 @@
 import { colors, NlTestedOverallValue, TimeframeOption, TimeframeOptionsList } from '@corona-dashboard/common';
-import { useState } from 'react';
 import { GgdTesten } from '@corona-dashboard/icons';
-import { css } from '@styled-system/css';
 import { GetStaticPropsContext } from 'next';
-import { Box } from '~/components/base';
-import { RadioGroup } from '~/components/radio-group';
-import { BoldText } from '~/components/typography';
+import { useState } from 'react';
+import { ChartTile, ChoroplethTile, Divider, DynamicChoropleth, InView, Markdown, PageInformationBlock, TileList, TimeSeriesChart } from '~/components';
```
Comment (VWSCoronaDashboard21) [inline packages/app/src/pages/landelijk/positief-geteste-mensen.tsx:5]: As mentioned in the Code conventions, we aim to stop using barrel files.

File: packages/app/src/pages/veiligheidsregio/[code]/positief-geteste-mensen.tsx [hunk 2]
```diff
@@ -1,14 +1,12 @@
 import { colors, TimeframeOption, TimeframeOptionsList } from '@corona-dashboard/common';
 import { GgdTesten } from '@corona-dashboard/icons';
-import { css } from '@styled-system/css';
 import { GetStaticPropsContext } from 'next';
 import { useRouter } from 'next/router';
 import { useState } from 'react';
-import { Box } from '~/components/base';
+import { ChartTile, ChoroplethTile, Divider, DynamicChoropleth, InView, Markdown, PageInformationBlock, TileList, TimeSeriesChart } from '~/components';
```
Comment (VWSCoronaDashboard21) [inline packages/app/src/pages/veiligheidsregio/[code]/positief-geteste-mensen.tsx:6]: As mentioned in the Code conventions, we aim to stop using barrel files.

**Review by VWSCoronaDashboard21 — CHANGES_REQUESTED** [review]

**Review by VWSCoronaDashboard21 — DISMISSED** [review]
LGTM!

File: packages/app/src/pages/landelijk/positief-geteste-mensen.tsx [hunk 3]
```diff
@@ -1,27 +1,32 @@
 import { colors, NlTestedOverallValue, TimeframeOption, TimeframeOptionsList } from '@corona-dashboard/common';
```
Comment (VWSCoronaDashboard18) [inline packages/app/src/pages/landelijk/positief-geteste-mensen.tsx:1]: We don't use barrel files anymore. Maybe split this line in separate lines.

File: packages/app/src/pages/landelijk/positief-geteste-mensen.tsx [hunk 4]
```diff
@@ -1,27 +1,32 @@
...
 import { thresholds } from '~/components/choropleth/logic/thresholds';
+import { Divider } from '~/components/divider';
+import { InView } from '~/components/in-view';
+import { Markdown } from '~/components/markdown';
+import { PageInformationBlock } from '~/components/page-information-block';
+import { TileList } from '~/components/tile-list';
+import { TimeSeriesChart } from '~/components/time-series-chart';
+import { BoldText } from '~/components/typography';
 import { Layout, NlLayout } from '~/domain/layout';
 import { GNumberBarChartTile, InfectedPerAgeGroup } from '~/domain/tested';
```
Comment (VWSCoronaDashboard18) [inline packages/app/src/pages/landelijk/positief-geteste-mensen.tsx:19]: See earlier comment about the barrel files.

File: packages/app/src/pages/landelijk/positief-geteste-mensen.tsx [hunk 5]
```diff
@@ -1,27 +1,32 @@
...
 import { useIntl } from '~/intl';
 import { Languages, SiteText } from '~/locale';
 import { ElementsQueryResult, getElementsQuery, getTimelineEvents } from '~/queries/get-elements-query';
 import { getArticleParts, getPagePartsQuery } from '~/queries/get-page-parts-query';
 import { createGetStaticProps, StaticProps } from '~/static-props/create-get-static-props';
-import { createGetChoroplethData, createGetContent, getLastGeneratedDate, selectNlData, getLokalizeTexts } from '~/static-props/get-data';
+import { createGetChoroplethData, createGetContent, getLastGeneratedDate, getLokalizeTexts, selectNlData } from '~/static-props/get-data';
 import { ArticleParts, PagePartQueryResult } from '~/types/cms';
 import { replaceComponentsInText, replaceVariablesInText, useReverseRouter } from '~/utils';
```
Comment (VWSCoronaDashboard18) [inline packages/app/src/pages/landelijk/positief-geteste-mensen.tsx:27]: See earlier comment about the barrel files.. And this goes for the whole PR...

File: packages/app/src/pages/landelijk/positief-geteste-mensen.tsx (hunk 3 above)
Comment (VWSCoronaDashboard26) [reply packages/app/src/pages/landelijk/positief-geteste-mensen.tsx:1]: These imports come from an entirely different package, meaning we would have to adjust the loader to facilitate this. I would personally argue that that is beyond the scope of this ticket and should be picked up in its own ticket if desired. What do you think?

File: packages/app/src/pages/landelijk/positief-geteste-mensen.tsx (hunk 4 above)
Comment (VWSCoronaDashboard26) [reply packages/app/src/pages/landelijk/positief-geteste-mensen.tsx]: Appropriate changes have been applied. Please feel free to resolve this conversation if approved.

File: packages/app/src/pages/landelijk/positief-geteste-mensen.tsx (hunk 5 above)
Comment (VWSCoronaDashboard26) [reply packages/app/src/pages/landelijk/positief-geteste-mensen.tsx]: Appropriate changes have been applied. Please feel free to resolve this conversation if approved.

File: packages/app/src/pages/landelijk/positief-geteste-mensen.tsx (hunk 3 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/pages/landelijk/positief-geteste-mensen.tsx:1]: Technically I think it is a barrel file, but I'm not sure if we would want to do it this way... But then the pros and cons wouldn't change with importing from outside the current package...
For now, I think it's up to you to adjust or not. And I would be fine with either one of them. And you could resolve this when you choose not to do it. Or let me now when you choose the first option and adjusted it.

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Great PR!

**Review by VWSCoronaDashboard28 — APPROVED** [review]

**Review by VWSCoronaDashboard27 — APPROVED** [review]

**Pull request merged**